newsModel.js

```diff
@@ -0,0 +1,22 @@
+class NewsModel{
+
+  constructor(){
+    this.headlines = []
+  }
+
+  getHeadlines(){
+    return this.headlines;
+  }
+
+  addHeadline(text) {
+    this.headlines.push(text)
+  }
+
+  setHeadlines(headlines) {
+    this.headlines = [...headlines]
+  }
+
```

Comment:
^This looks so neat and tidy, looking forward to our debrief for a chat about this!